packages/docs-ui/src/controllers/render-controllers/doc.render-controller.ts

```diff
@@ -76,8 +76,6 @@ export class DocRenderController extends RxDisposable implements IRenderModule {
             top: 0,
             bottom: 0,
             right: 0,
-            explicitViewportWidthSet: false,
```

Comment:
why  delete code in doc render

Reply:
viewport do not need this config anymore